### ISSUE TITLE
Couple suggestions: Add texture default pack variants of the guards and a ranking system.

### ISSUE THREAD

**Issue by TootyCappy** [seymourimadeit/guardvillager-suggestions#132]
I think it would be really cool to add variant textures for the guards. You can still equip them with armor, but I think it would be cool if they have their different style of suits. (ex. below, most of them are from bedrock).
![image](https://user-images.githubusercontent.com/123497768/214376955-7c4c3f8a-f2fc-49db-bb41-c191deb7f401.png)
![image](https://user-images.githubusercontent.com/123497768/214377044-9066f91d-8fb1-4c70-9f3c-d4c8003f527d.png)
![image](https://user-images.githubusercontent.com/123497768/214381289-a82006a2-3c6b-4d2e-8ea5-0ca4922ef208.png)
![image](https://user-images.githubusercontent.com/123497768/214378757-a2f29321-24e9-4cb7-9072-0e42a11f788e.png)
thubusercontent.com/123497768/214376696-1eff6ded-c94e-4a4c-bdbc-a86ad1abdfcf.png)
![image](https://user-images.githubusercontent.com/123497768/214382100-af863e86-f58b-4525-8577-bc4942b1012f.png)
![image](https://user-images.githubusercontent.com/123497768/214382424-f57ef5ec-75d6-431d-8612-2fd991d00525.png)
![image](https://user-images.githubusercontent.com/123497768/214385310-01b1cd55-ad91-4bac-98f5-21e236d5ef5b.png)

As for the ranking system, you can promote them to ranks like a commander, captain, and general. When they reach their ranking, they would have certain abilities like boosting morale, a power that calls for a blessing to renew himself, you, and other guards to full health and physically boost their attributes temporarily when he is near death, and call for reinforcements using the horn.) When the guards survived the raid, they automatically rank up. But you can manually rank them by granting exp. points or use an item like a totem of undying maybe? Sorry if it sounds too much. I thought it would be cool!

**Comment by omeggga** [comment]
I think this armor system would be cool, as the current armor system makes them look very bulky when donning it, not unlike the player.

**Comment by TootyCappy** [comment]
> I think this armor system would be cool, as the current armor system makes them look very bulky when donning it, not unlike the player.

Yeah. If this mod really wants to reward players for helping villages, they should be granted some sort of complex command system similar to the recruit mod.

**Comment by omeggga** [comment]
I don't think we need to go quite that far to be honest (though looking at it now it is pretty cool). Though crossbowmen do need a massive improvement, as it stands they're [absolute garbage](https://youtu.be/CJKeTYO6OsA?t=63).

**Comment by TootyCappy** [comment]
> I don't think we need to go quite that far to be honest (though looking at it now it is pretty cool). Though crossbowmen do need a massive improvement, as it stands they're [absolute garbage](https://youtu.be/CJKeTYO6OsA?t=63).

Yeah, imagine if they have muskets :/. There should be a data pack that helps with friendly fire.

**Comment by omeggga** [comment]
> Yeah, imagine if they have muskets :/. There should be a data pack that helps with friendly fire.

Config already allows villagers to take no damage from FF. But what I mean is crossbowmen should be made to fire more often. As it is now they spend half a minute liing up their shots.
